src/types/onyx/TransactionViolation.ts

```diff
@@ -31,7 +31,7 @@ type TransactionViolation = {
     };
 };
 
-type TransactionViolations = Record<string, TransactionViolation>;
+type TransactionViolations = TransactionViolation[];
```

Comment:
Are you sure about this change?

Reply:
This type is used in TransactionViolations [collection](https://github.com/VickyStash/Expensify/blob/ts-migration/iou/src/ONYXKEYS.ts#L465). And a collection of TransactionViolations is a collection of arrays, not objects
![image](https://github.com/Expensify/App/assets/23176449/da733c20-2115-4b77-bc5a-758f5cd961fb)

Reply:
This change is correct. Good catch!